app/utils.py

```diff
@@ -102,7 +104,8 @@ def format_resource_search(hit):
     return formatted
 
 
-def standardize_response(payload={}, status_code=200):
+@versioned(throw_on_invalid=False)
+def standardize_response(payload={}, status_code=200, version=0.0):
```

Comment:
I haven't ran the code yet (I'm about to do that next), so maybe this isn't what it seems. Is it possible though that under some circumstance the API would return `"apiVersion": "0.0"` in a response?

We want to default to the most recent version, not 0.0, so would it be possible to import that value and pop that in as the default value for `version` here?

Reply:
It will never actually use that. That is just present so that I can place it at the end of the parameter list in case you are using positional rags elsewhere in the code base. The decorator will always inject a real value, defaulting to the latest version.

Reply:
Can we just leave a default off or would that break things?

Reply:
It's required, so that I don't break other parts of code that expects the preceding arguments to maintain their current ordering. That said, I'll change it to point at the latest version to make it look less confusing.

- [ ] set default value of standardize_response version parameter to latest API version